### DISCUSSION TITLE
Allow for custom class in ajax forms

### DISCUSSION THREAD

**Discussion by knynkwl** [solspace/craft-freeform#233] [Questions & Answers] [answered]
Currently, it looks like we can update content (error/success message, etc) through a data attribute but it would be great if the same was true for classes.

<img width="372" alt="Screen Shot 2021-11-10 at 8 15 07 AM" src="https://user-images.githubusercontent.com/4225771/141119959-775cf712-9ad3-4a7a-bfb3-319c5791c08d.png">
<img width="637" alt="Screen Shot 2021-11-10 at 8 16 51 AM" src="https://user-images.githubusercontent.com/4225771/141119985-83f0885e-d352-4b00-aeb4-7bcaac876161.png">

**Comment by gustavs-gutmanis** [comment, marked as answer]
This is just getting the defaults internally, for freeform. You shouldn't use these, you should instead use the events exposed by freeform to manipulate what you want.

[Here's a link to the docs](https://docs.solspace.com/craft/freeform/v3/developer/js-plugin.html#plugin-options) explaining how to adjust the options that you have mentioned.

Regarding the feature you requested, could you please elaborate a bit more? What did you mean by classes?
